extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -580,7 +580,10 @@ export class ProjectsController {
 			} else if ((<ISystemDatabaseReferenceSettings>settings).systemDb !== undefined) {
 				await project.addSystemDatabaseReference(<ISystemDatabaseReferenceSettings>settings);
 			} else {
-				await project.addDatabaseReference(<IDacpacReferenceSettings>settings);
+				// update dacpacFileLocation to relative path to project file
+				const dacpacRefSettings = settings as IDacpacReferenceSettings;
+				dacpacRefSettings.dacpacFileLocation = vscode.Uri.file(path.relative(project.projectFolderPath, dacpacRefSettings.dacpacFileLocation.fsPath));
```

Comment:
do we have a way to detect if its different drive for windows and give appropriate error there...

Reply:
added screenshot of the error message to the PR description

Reply:
for all. Are there any concerns for having it not be specifically for windows?

Reply:
no not really, was not sure if that's needed for others is all. didnt notice the issue with any other.